groups/chips-rds/variables.tf

```
@@ -149,3 +149,21 @@ variable "auto_minor_version_upgrade" {
   description = "True/False value to allow AWS to apply minor version updates to RDS without approval from owner"
   default     = true
 }
+
+# ------------------------------------------------------------------------------
+# RDS CloudWatch Alarm Variables
+# ------------------------------------------------------------------------------
+variable "alarm_actions_enabled" {
+  type        = string
```

The type here should be `bool` (see the [module docs](https://github.com/companieshouse/terraform-modules/tree/main/aws/rds_cloudwatch_alarms)):

```suggestion
  type        = bool
```

---

Updated in latest commit